src/test/java/com/klarna/hiverunner/AggregateViewTest.java

```diff
@@ -0,0 +1,30 @@
+package com.klarna.hiverunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.klarna.hiverunner.annotations.HiveSQL;
+
+@RunWith(StandaloneHiveRunner.class)
```

Comment:
Going forward I think all new tests should use JUnit5 and the extension instead of the Runner, can you make it so here please?